docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.5"
+
+services:
+  website:
+    env_file: .env
+    ports:
+      - ${WEBSITE_EXPOSED_PORT-8020}:${WEBSITE_CONTAINER_PORT-3000}
+    volumes:
+      - .:/code
+    build:
+      dockerfile: Dockerfile-dev
+      context: .
+    networks:
+      -  faf-stack
+    
+networks:
+  faf-stack:
+    name: faf-stack_faf
```

Comment:
I wouldn't call it faf_stack since realistically that is dying soon TM

Reply:
docker needs it like this, otherwise i cannot access the containers from the faf-stack.
more info: https://stackoverflow.com/questions/38088279/communication-between-multiple-docker-compose-projects

Reply:
got it didn't realize this was supposed to be used in conjunction with faf_stack